obspy/io/nordic/core.py

```diff
@@ -800,23 +799,27 @@ def readwavename(sfile, encoding='latin-1'):
     :rtype: list
     """
     with open(sfile, 'r', encoding=encoding) as f:
-        wavenames = _readwavename(f=f)
+        tagged_lines = _get_line_tags(f=f)
+        if len(tagged_lines['6']) == 0:
+            msg = ('No waveform files in sfile %s' % sfile)
+            warnings.warn(msg)
+        wavenames = _readwavename(tagged_lines=tagged_lines['6'])
     return wavenames
 
 
-def _readwavename(f):
+def _readwavename(tagged_lines):
     """
     Internal wave-name reader.
 
-    :type f: file
-    :param f: File open in read-mode
+    :type tagged_lines: list
+    :param tagged_lines: 
```

Comment:
Trailing whitespace here makes flake8 fail

Reply:
sorry, should have checked flake8. Fixed now.

Reply:
No big deal and thanks.